src/Filters/atoms/DateTimeBlock.tsx

```diff
@@ -144,17 +144,23 @@ const DateTimeBlock : React.FC<IProps> = ({
   }, [date, displayMinutes, setDateCallback]);
 
   const setDateMinutes = useCallback(({target: {value}}: React.ChangeEvent<HTMLInputElement>) => {
-    const minuteRegex = /^[0-5]{0,1}[0-9]{0,1}$/;
-    if (!minuteRegex.test(value)) {
-      return;
```

Comment:
Removing this regular expression check will allow to add decimal and 'e'.
Please check and update Regular expression if needed.

![image](https://user-images.githubusercontent.com/95204740/226606370-1422b52a-c528-4593-ae22-106ba2955787.png)

Reply:
Thanks for the feedback. Done